forest/src/cli/fetch_params_cmd.rs

```diff
@@ -32,7 +33,11 @@ impl FetchCommands {
             panic!("Sector size option must be chosen. Choose between --all, --keys, or <size>");
         };
 
-        get_params_default(sizes, self.verbose).await.unwrap();
+        set_dir_env(config.data_dir.clone());
```

Comment:
There's no need to set the environment here in the fetch command. It's only used in the forest daemon.